From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: guihkx <guihkx@users.noreply.github.com>
Date: Sun, 24 Feb 2019 16:53:17 -0300
Subject: Remove stupid startup delay

---
 ArchiSteamFarm/Core/ASF.cs | 1 -
 1 file changed, 1 deletion(-)

diff --git a/ArchiSteamFarm/Core/ASF.cs b/ArchiSteamFarm/Core/ASF.cs
index 9d4b3ae56..6af72c34f 100644
--- a/ArchiSteamFarm/Core/ASF.cs
+++ b/ArchiSteamFarm/Core/ASF.cs
@@ -709,7 +709,6 @@ private static async Task RegisterBots() {
 				return;
 			case > MaximumRecommendedBotsCount:
 				ArchiLogger.LogGenericWarning(string.Format(CultureInfo.CurrentCulture, Strings.WarningExcessiveBotsCount, MaximumRecommendedBotsCount));
-				await Task.Delay(SharedInfo.InformationDelay).ConfigureAwait(false);
 
 				break;
 		}
-- 
2.46.0

