From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: guihkx <626206+guihkx@users.noreply.github.com>
Date: Fri, 15 Dec 2023 12:29:55 -0300
Subject: Disallow the detection of custom plugins

---
 ArchiSteamFarm/Plugins/PluginsCore.cs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ArchiSteamFarm/Plugins/PluginsCore.cs b/ArchiSteamFarm/Plugins/PluginsCore.cs
index 6d012a2bd..145e4a8b5 100644
--- a/ArchiSteamFarm/Plugins/PluginsCore.cs
+++ b/ArchiSteamFarm/Plugins/PluginsCore.cs
@@ -54,7 +54,7 @@
 namespace ArchiSteamFarm.Plugins;
 
 public static class PluginsCore {
-	internal static bool HasCustomPluginsLoaded => ActivePlugins.Any(static plugin => plugin is not OfficialPlugin officialPlugin || !officialPlugin.HasSameVersion());
+	internal static bool HasCustomPluginsLoaded => false;
 
 	[ImportMany]
 	internal static FrozenSet<IPlugin> ActivePlugins { get; private set; } = FrozenSet<IPlugin>.Empty;
-- 
2.50.1

