From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: guihkx <626206+guihkx@users.noreply.github.com>
Date: Fri, 15 Dec 2023 12:29:55 -0300
Subject: Disallow the detection of custom plugins

---
 ArchiSteamFarm/Plugins/PluginsCore.cs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ArchiSteamFarm/Plugins/PluginsCore.cs b/ArchiSteamFarm/Plugins/PluginsCore.cs
index e91b1665..b914707a 100644
--- a/ArchiSteamFarm/Plugins/PluginsCore.cs
+++ b/ArchiSteamFarm/Plugins/PluginsCore.cs
@@ -58,7 +58,7 @@ public static class PluginsCore {
 	[PublicAPI]
 	public static int ActivePluginsCount => ActivePlugins.Count;
 
-	internal static bool HasCustomPluginsLoaded => ActivePlugins.Any(static plugin => plugin is not OfficialPlugin officialPlugin || !officialPlugin.HasSameVersion());
+	internal static bool HasCustomPluginsLoaded => false;
 
 	[ImportMany]
 	internal static FrozenSet<IPlugin> ActivePlugins { get; private set; } = FrozenSet<IPlugin>.Empty;
-- 
2.44.0

