From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: guihkx <guihkx@users.noreply.github.com>
Date: Sun, 24 Feb 2019 16:53:17 -0300
Subject: Remove stupid startup delay

---
 ArchiSteamFarm/Core/ASF.cs | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/ArchiSteamFarm/Core/ASF.cs b/ArchiSteamFarm/Core/ASF.cs
index 173b4973a..256048e71 100644
--- a/ArchiSteamFarm/Core/ASF.cs
+++ b/ArchiSteamFarm/Core/ASF.cs
@@ -53,9 +53,6 @@
 namespace ArchiSteamFarm.Core;
 
 public static class ASF {
-	// This is based on internal Valve guidelines, we're not using it as a hard limit
-	private const byte MaximumRecommendedBotsCount = 10;
-
 	[PublicAPI]
 	public static readonly ArchiLogger ArchiLogger = new(SharedInfo.ASF);
 
@@ -491,10 +488,6 @@ private static async Task OnCreatedJsonFile(string name, string fullPath) {
 			await bot.OnConfigChanged(false).ConfigureAwait(false);
 		} else {
 			await Bot.RegisterBot(botName).ConfigureAwait(false);
-
-			if (Bot.Bots.Count > MaximumRecommendedBotsCount) {
-				ArchiLogger.LogGenericWarning(Strings.FormatWarningExcessiveBotsCount(MaximumRecommendedBotsCount));
-			}
 		}
 	}
 
@@ -690,11 +683,6 @@ private static async Task RegisterBots() {
 				ArchiLogger.LogGenericWarning(Strings.ErrorNoBotsDefined);
 
 				return;
-			case > MaximumRecommendedBotsCount:
-				ArchiLogger.LogGenericWarning(Strings.FormatWarningExcessiveBotsCount(MaximumRecommendedBotsCount));
-				await Task.Delay(SharedInfo.InformationDelay).ConfigureAwait(false);
-
-				break;
 		}
 
 		await Utilities.InParallel(botNames.OrderBy(static botName => botName, Bot.BotsComparer).Select(Bot.RegisterBot)).ConfigureAwait(false);
-- 
2.50.0

