From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: guihkx <guihkx@users.noreply.github.com>
Date: Sat, 3 Jul 2021 13:24:31 -0300
Subject: [PATCH] Lower AutoSteamSaleEvent interval from 8 to 3 hours

---
 ArchiSteamFarm/Steam/Integration/SteamSaleEvent.cs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ArchiSteamFarm/Steam/Integration/SteamSaleEvent.cs b/ArchiSteamFarm/Steam/Integration/SteamSaleEvent.cs
index ecafa6fd..724550d4 100644
--- a/ArchiSteamFarm/Steam/Integration/SteamSaleEvent.cs
+++ b/ArchiSteamFarm/Steam/Integration/SteamSaleEvent.cs
@@ -46,7 +46,7 @@ internal sealed class SteamSaleEvent : IAsyncDisposable {
 			ExploreDiscoveryQueue,
 			null,
 			TimeSpan.FromHours(1.1) + TimeSpan.FromSeconds(ASF.LoadBalancingDelay * Bot.Bots?.Count ?? 0), // Delay
-			TimeSpan.FromHours(8.1) // Period
+			TimeSpan.FromHours(3.1) // Period
 		);
 	}
 
-- 
2.25.1

